examples/lambda-functions/get-by-id.ts

```diff
@@ -0,0 +1,91 @@
+import { APIGatewayProxyEvent, APIGatewayProxyResult, Context } from 'aws-lambda';
+import { Metrics } from '@aws-lambda-powertools/metrics';
+import { Logger } from '@aws-lambda-powertools/logger';
+import { Tracer } from '@aws-lambda-powertools/tracer';
+import { DynamoDB } from 'aws-sdk';
+
+// Create the PowerTools clients
+const metrics = new Metrics();
+const logger = new Logger();
+const tracer = new Tracer();
+
+// Create DynamoDB DocumentClient and patch it for tracing
+const docClient = tracer.captureAWS(new DynamoDB.DocumentClient());
+
+// Get the DynamoDB table name from environment variables
+const tableName = process.env.SAMPLE_TABLE;
+
+/**
+ *
+ * Event doc: https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html#api-gateway-simple-proxy-for-lambda-input-format
+ * @param {Object} event - API Gateway Lambda Proxy Input Format
+ *
+ * Return doc: https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html
+ * @returns {Object} object - API Gateway Lambda Proxy Output Format
+ *
+ */
+
+
+export const getByIdHandler = async (event: APIGatewayProxyEvent, context: Context): Promise<APIGatewayProxyResult> => {
+    if (event.httpMethod !== 'GET') {
+        throw new Error(`getById only accepts GET method, you tried: ${event.httpMethod}`);
+    }
+    // Tracer: Get facade segment created by AWS Lambda
+    const segment = tracer.getSegment();
+
+    // Tracer: Create subsegment for the function & set it as active
+    const handlerSegment = segment.addNewSubsegment(`## ${process.env._HANDLER}`);
+    tracer.setSegment(handlerSegment);
+
+    // Tracer: Annotate the subsegment with the cold start & serviceName
+    tracer.annotateColdStart();
+    tracer.addServiceNameAnnotation();
+
+    // Tracer: Add annotation for the awsRequestId
+    tracer.putAnnotation('awsRequestId', context.awsRequestId);
+
+    // Metrics: Capture cold start metrics
+    metrics.captureColdStartMetric();
+
+    // All log statements are written to CloudWatch
+    logger.debug('received:', event);
+
+    // Get id from pathParameters from APIGateway because of `/{id}` at template.yaml
+    const id = event.pathParameters!.id;
+
+    // Get the item from the table
+    // https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/DynamoDB/DocumentClient.html#get-property
+    var params = {
```

Comment:
```suggestion
    const params = {
```
Can we use `const` to be consistent?

Reply:
fixed